src/DrupalCheck.php

```diff
@@ -33,9 +33,16 @@ public static function getConfigurableOptions(): OptionsResolver
   {
       $resolver = new OptionsResolver();
       $resolver->setDefaults([
-        'drupal_root' => null,
+        'drupal_root' => NULL,
+        'deprecations' => NULL,
+        'analysis' => NULL,
+        'php8' => NULL,
       ]);
       $resolver->addAllowedTypes('drupal_root', ['string', 'null']);
+      $resolver->addAllowedTypes('deprecations', ['boolean', true]);
```

Comment:
My bad, this wasn't the place to add the default values.

```suggestion
      $resolver->addAllowedTypes('deprecations', ['boolean', 'null']);
```

Reply:
Updated default and allowed values in another commit https://github.com/Metadrop/grumphp-drupal-check/pull/9/commits/b675fc4e51e0d3a5d19de169c87fe64b9d0465fa